recipes/lmdb/Makefile.patch

```diff
@@ -0,0 +1,13 @@
+diff --git libraries/liblmdb/Makefile libraries/liblmdb/Makefile
+index 0940c49..2550f1d 100644
+--- libraries/liblmdb/Makefile
++++ libraries/liblmdb/Makefile
+@@ -26,7 +26,7 @@ OPT = -O2 -g
+ CFLAGS	= $(THREADS) $(OPT) $(W) $(XCFLAGS)
+ LDLIBS	=
+ SOLIBS	=
+-prefix	= /usr/local
++prefix	=
```

Comment:
There is no configure step and there didn't seem to be a way to tell it not to use this prefix so we had to patch it out. Otherwise, everything gets installed at `$DESTDIR/usr/local`, which we don't really want. We simply want to set `$DESTDIR` and have everything installed there.